HOWTO.md

```diff
@@ -92,7 +92,7 @@ Some thoughts to consider when adding a new message:
 It's highly recommended to use the docker container to run the release process,
 the docker container can be pulled from DockerHub and launched via this command:
 
-``docker run  -v $PWD:/mnt/workspace -i -t swiftnav/libsbp-build:2021.04.22``
```

Comment:
Should we keep this change?

Reply:
Yeah can do. What do you want to do about `2021.04.22` - the only change in that image was to include `clang-format` but since it's not being used... Just keep using `2021.04.22` anyway since it's the latest?

Reply:
Yes, let's keep the latest, and when clang-format is reintroduced we won't need to worry about this

Reply:
Done